internal/util/man/man_test.go

```diff
@@ -155,7 +155,7 @@ Java server ConfigMap
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: java\-config
```

Comment:
Is this change related?

Reply:
This relates to the upgrade of go-md2man to v2.0.1.
I was not able to figure out how to prevent go from upgrading it so I went all in and fixed the test. @droot reviewed this in oci branch (#2669)